cmd/rancher-desktop-guestagent/main.go

```diff
@@ -36,9 +39,15 @@ var (
 	configPath       = flag.String("kubeconfig", "/etc/rancher/k3s/k3s.yaml", "path to kubeconfig")
 	enableIptables   = flag.Bool("iptables", true, "enable iptables scanning")
 	enableKubernetes = flag.Bool("kubernetes", false, "enable Kubernetes service forwarding")
+	enableDocker     = flag.Bool("docker", false, "enable Docker event monitoring")
 )
 
-const iptablesUpdateInterval = 3 * time.Second
+const (
+	iptablesUpdateInterval   = 3 * time.Second
+	dockerSocketInterval     = 5 * time.Second
+	dockerSocketRetryTimeout = 2 * time.Minute
+	dockerSocketFile         = "/var/run/docker.sock"
```

Comment:
This seems like a thing that might need to be customizable (but I guess we don't have need for it for now)